superset/models/slice.py

```diff
@@ -286,14 +286,14 @@ def get_query_context(self) -> Optional[QueryContext]:
 
     def get_explore_url(
         self,
-        base_url: str = "/superset/explore",
+        base_url: str = "/explore",
         overrides: Optional[Dict[str, Any]] = None,
     ) -> str:
         overrides = overrides or {}
         form_data = {"slice_id": self.id}
         form_data.update(overrides)
```

Comment:
Not sure if `overrides` will be recognized by the new explore endpoint, since it doesn't handle `form_data` param